wa/commands/runstatus.py

```diff
@@ -0,0 +1,298 @@
+import os
+import logging
+from datetime import datetime, timedelta
+
+import colorama
+
+from wa.framework.configuration.core import Status
```

Comment:
These should be in alphabetical order. In `nvim`:
1. `<shift>-v` to enter line-select mode.
2. Move down to select all import lines below, then
3. `:sort<cr>`